misc/docs/source/tutorial/code_snippets/basemap_plot_custom.py

```diff
@@ -0,0 +1,36 @@
+from mpl_toolkits.basemap import Basemap
+import numpy as np
+import matplotlib.pyplot as plt
+
+from obspy import read_inventory, read_events
+
+# Set up a custom basemap, example is taken from basemap users' manual
+fig, ax = plt.subplots()
+
+# setup albers equal area conic basemap
+# lat_1 is first standard parallel.
+# lat_2 is second standard parallel.
+# lon_0, lat_0 is central point.
+m = Basemap(width=8000000, height=7000000,
```

Comment:
Just copy pasted from the basemap example.. in any case, width/height is in projection units. Not sure but I'm guessing meters.